lib/membership/member.js

```diff
@@ -242,4 +217,26 @@ Member.isStatusPingable = function isStatusPingable(status) {
     }
 };
 
+/**
```

Comment:
There is actually not a lot of documentation in the project. I could change it to be consistent but the nice thing about this that some IDE's actually use it and warn the user in case of misuse. In the feature we could actually generate documentation based on it (see http://usejsdoc.org/ for more info)